### ISSUE TITLE
MenuComposer error 

### ISSUE THREAD

**Issue by amerhamdan3** [LaravelDaily/Laravel-Demo-Courses-Enrollment#6]
https://github.com/LaravelDaily/Laravel-Demo-Courses-Enrollment/blob/a21b9838dce4bc6b81b3b374fcddfa538a32039c/app/Http/View/Composers/MenuComposer.php#L9

this should be MenuComposer 
or it will give error 

and we have another file called MenuComposer with wrong namespace but i think we will not use it in this project

**Comment by dalpat** [comment]
@amerhamdan3 You are right. I've renamed it to MenuComposer and it worked. The issue can be marked as solved I think.
